.goreleaser.yml

```diff
@@ -1,6 +1,7 @@
 project_name: datamon
 before:
   hooks:
+    - go get -u github.com/gobuffalo/packr/v2/packr2
```

Comment:
No this won't work:
- goreleaser will detect that the git tree is not clean and will refuse to release

The only way, AFAIK, is to commit the generated code.

Reply:
Good to know. I was debating the same choice, it seems to be the cleanest.

Reply:
in case we're looking for additional options,

using the `Makefile` to drive packr, including and `git` setup and teardown necessary to satisfy goreleaser, both in CI and on the local, is also possible.  the advantage here is single source of truth.